packages/xchain-bitcoin/src/client.ts

```diff
@@ -52,6 +51,11 @@ class Client implements BitcoinClient, XChainClient {
   nodeUrl = ''
   nodeApiKey = ''
 
+  derive_path = {
+    mainnet: "84'/0'/0'/0/0",
+    testnet: "84'/1'/0'/0/0",
+  }
+
```

Comment:
Let keep it private: `private derive_path = {...}` or define `export const DERIVE_PATH = `{...}` outside from `Client`